### ISSUE TITLE
Is it possible to drop make file and use tox only?

### ISSUE THREAD

**Issue by jcchavezs** [hypertrace/pythonagent#156]
We introduced makefile to be able to abstract the tasks during development workflow, however many of the commands are just a mimic of the tox tool. We should explore the possibility to use tox only.

**Comment by rcbjBlueMars** [comment]
Yes. Experimented with having a top-level tox run the tox configurations throughout the integration tests subdirectories. Ran into some issues with it. Didn't want to get bogged down with it early in the development cycle.

**Comment by shril** [comment]
Looking into this @jcchavezs
